src/compiler/parser.ts

```diff
@@ -8729,7 +8778,8 @@ namespace Parser {
                     state = JSDocState.BeginningOfLine;
                     indent = 0;
                 }
-                loop: while (true) {
+                loop:

```

Comment:
I was going to say "not sure how I feel about it" until I realized that's how labels are formatted in Go and I feel like that's ok. zig does it the way we had before. The new method at least doesn't visually ambiguate with an object literal or var+type annotation so that feels like a win-ish.

Not sure there's a toggle for that in dprint or what anyway. Labels are super rare.

Reply:
Opened https://github.com/dprint/dprint-plugin-typescript/pull/538 to maintain the position for now. We can make this configurable in the future.

Reply:
I don't think we actually need it if you don't want to add it; I think the new way is better (though that's an opinion).